ios/FluentUI/Core/Theme/Tokens/ShadowInfo.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import CoreGraphics
+
+/// Represents a two-part shadow as used by FluentUI.
+public struct ShadowInfo {
```

Comment:
I don't think any of the tokens you're moving with this change use this, so we don't need to bring it over right now, either.

Reply:
I would have to also remove all the alias tokens that reference `shadow`, which I would rather leave in to keep it all together for now.